docling/backend/docling_parse_v4_backend.py

```
@@ -153,40 +208,67 @@ def page_count(self) -> int:
     def is_valid(self) -> bool:
         return self.page_count() > 0
 
     def unload(self):
         super().unload()
+        # Shut down the persistent worker process
+        if self._worker_alive:
+            try:
+                self._req_queue.put(None)  # sentinel
+                self._worker.join(timeout=5)
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)